x-pack/plugins/maps/public/classes/sources/es_pew_pew_source/es_pew_pew_source.js

```diff
@@ -6,7 +6,8 @@
 
 import React from 'react';
 import uuid from 'uuid/v4';
-import turf from 'turf';
+import bbox from '@turf/bbox';
+import { multiPoint } from '@turf/helpers';
```

Comment:
same nit-picky question as above. should we rename these functions like `turfBbox` and `turfMultipoint`?